### ISSUE TITLE
Spacing seems wrong in the morse code

### ISSUE THREAD

**Issue by ncw** [kenkawakenkenke/MorseNotification#2]
When I receive a whatsapp notification I get the morse buzzing which is great. However the timing seems off and I find it very difficult to read.

I recorded it so you can see what I mean (I didn't catch the start of the notification)

[morse-notification.zip](https://github.com/kenkawakenkenke/MorseNotification/files/7713373/morse-notification.zip)

In audacity it looks like this

![image](https://user-images.githubusercontent.com/536803/146043374-6e247e8c-0db2-4f08-bf99-a12d07668ac0.png)

See how the spacing between the dots and dashes isn't consistent and thelength of the dashes in particular are quite variable. It sounds to me like someone sending on a straight key which is not what I expected for computer generated morse!

 I put the letters one on top of each other so you can see what I mean more easily.

![image](https://user-images.githubusercontent.com/536803/146045841-968a29ce-622b-408a-80bd-17f9f876b63a.png)

This is on a Google Pixel 5 with Android 11.

73 de 2E0WPM

**Comment by kenkawakenkenke** [comment]
Thanks for the detailed report!
I'm expecting this to be a limitation of the vibration hardware which wasn't designed for super precise vibration durations, so the fix isn't going to be easy.
Let me keep this issue open though to remind myself in case I think of some way to alleviate this.

**Comment by ncw** [comment]
Yes I think you are probably right about the hardware. Unfortunately I can't decode the Morse it sends at the moment (I'm not super experienced at Morse but I should be able to decode Morse at this speed (its about 12wpm I think).
